Fix and rename edgeR process

JenkoB opened the pull request.

mstajdohar commented on resolwe_bio/processes/differential_expression/edgeR.yml:
```diff
@@ -89,7 +89,7 @@
       diffexp_filtering.R -counts 'counts.tab' -filter {{filter}} -out counts_filtered.tab
 
-      edger.R counts_filtered.tab --sampleConditions ${sampleConditions}
+      run_edger.R counts_filtered.tab --sampleConditions ${sampleConditions}
```
Do we have a test for this? Why didn't it fail for the wrong script name?

JenkoB commented:
I renamed the the tool to be consistent with names of other tools (like
run_deseq2, rubn_lima). I changed it in tool name and in the process, so
the process could not fail.

On Tue, Nov 8, 2016 at 8:44 AM, Miha Stajdohar <notifications@github.com>
wrote:

> *@mstajdohar* approved this pull request.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/genialis/resolwe-bio/pull/127#pullrequestreview-7568976>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI2mIyvgVoqyW3-UODrbf9eepY__3kZTks5q8ChOgaJpZM4KiiBn>
> .
>



-- 
mail: barbara.jenko@gmail.com <barbara.jenk@best.eu.org>
SloveniaTEL : +386 31283739
